### PR TITLE
Feature/ae ab read lists and items 2

### PR DESCRIPTION
**Pull request by arbitrary-berry**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Learned Firestore and accessed data while passing props. Rendered name of shopping list and name of each item within the list.

## Related Issue

closes 2

## Acceptance Criteria

 In Home.jsx, the SingleList component is used to render the name of each shopping list a user is subscribed to
 In List.jsx, the ListItem component is used to render the name of each item

## Type of Changes

feature add

### Before

Screen Shot 2024-02-06 at 5 32 10 PM

Screenshot 2024-02-06 at 7 33 33 PM

### After
Screen Shot 2024-02-06 at 5 35 34 PM

Screen Shot 2024-02-06 at 5 35 55 PM

## Testing Steps / QA Criteria

Sign into app, the user will now see the test-list on their homepage. 
Then go to /list and the items will appear on the screen.

### CODE REVIEW

**Review by flbarfield — APPROVED** [review]

**Review by jtkabenni — APPROVED** [review]
Hi, looks good. Two things to note:
- SingleList.jsx was updated too, though the issue notes says to just update Home and List. Should be fine, probably just helping our future selves out!
- When I test I see a bit of strange behavior: I go to /list and don't see the items. Then I go to home (/), click on the test-list button, and then after going to /list I can now see the items. Not sure if this is expected, just wanted to flag. Seems like no data is getting passed to the List.jsx initially.

Screenshot 2024-02-06 at 9 48 14 PM

**Comment by arbitrary-berry** [comment]
@jtkabenni 

- Thank you for the call out, you're right we did change it. Now, I'm curious how we could've completed the task without that change.

- Interesting there was that hiccup. @flbarfield Did you have a similar issue?

**Comment by bdanzer** [comment]
> Hi, looks good. Two things to note:
> 
> * SingleList.jsx was updated too, though the issue notes says to just update Home and List. Should be fine, probably just helping our future selves out!
> * When I test I see a bit of strange behavior: I go to /list and don't see the items. Then I go to home (/), click on the test-list button, and then after going to /list I can now see the items. Not sure if this is expected, just wanted to flag. Seems like no data is getting passed to the List.jsx initially.
> 
> Screenshot 2024-02-06 at 9 48 14 PM

@jtkabenni These are great questions! Also wanted to mention the way you are thinking is great! 

I think issue number #2  might not be worded correctly because it does say it wants the items to be rendered out but to get those items we needed to set the path. So not sure how to have rendered those items without setting the path. But it looks like issue #4 has quite a bit to do that will help fix some of the issues you called out.

Overall I would keep track of weird things you see through this project and would even suggest creating issues with weird behaviors you see. In a real job setting if I saw something weird I would bring it up in the slack channel and if people find it's weird too I would create an issue.

@arbitrary-berry I think this PR looks good for now, there will probably be more to fill in the gaps in future tickets but this seems to be a good start!

**Review by bdanzer — APPROVED** [review]
LGTM! #4  Will expand on the work done here so I think what is done so far is good and I think we met the criteria of #2

File: src/views/List.jsx [hunk 1]
```diff
@@ -12,6 +16,7 @@ export function List({ data }) {
 				 * using the `ListItem` component that's imported at the top
 				 * of this file.
 				 */}
+				{listItems}
```
Comment (bdanzer) [inline src/views/List.jsx:19]: @aberger3647 @arbitrary-berry small nit: intead of storing as a variable you could do this also:

```
return (
    <>
        <p>
           {data.map((data) => (
              <ListItem key={data.id} name={data.name} />
           ))}
        </p>
     </>
);
```

Either approach is fine and no need to change this, just wanted to point out how it's usually written.

Comment (aberger3647) [reply src/views/List.jsx:19]: Makes sense! No need to create a variable here. Thanks.

**Pull request merged**